src/easyreflectometry/project.py

```diff
@@ -132,14 +135,16 @@ def _default_info(self):
             modified=datetime.datetime.now().strftime('%d.%m.%Y %H:%M'),
         )
 
-    def create_project_dir(self):
-        if not os.path.exists(self._path):
-            os.makedirs(self._path)
-            os.makedirs(self._path / 'experiments')
+    def create(self):
+        if not os.path.exists(self.path):
+            os.makedirs(self.path)
+            os.makedirs(self.path / 'experiments')
+            self._created = True
+            self._timestamp_modification()
         else:
-            print(f'ERROR: Directory {self._path} already exists')
+            print(f'ERROR: Directory {self.path} already exists')
 
-    def save_project_json(self, overwrite=False):
+    def save_as_json(self, overwrite=False):
         if self.path_json.exists() and not overwrite:
```

Comment:
is this logic right? `overwrite` should be `True` to allow overwriting, yet here you're removing the json when `overwrite` is `False`.

Reply:
Yes, you are right!  Fixed